src/contexts/account/useWithdrawals.ts

```diff
@@ -72,62 +75,58 @@ export const useClaimingBot = (address: string | null) => {
 /**
  * Groups pending withdrawals that are within 5 minutes time span
  */
-const groupingTimeSpan: number = 5 * 60;
-const formatPendingWithdrawals = (values: string[], timestamps: string[]): PendingWithdrawal[] => {
+const groupingTimeSpan = 5n * 60n;
+const formatPendingWithdrawals = (values: bigint[], timestamps: bigint[]): PendingWithdrawal[] => {
   const sortedTimestamps = [...timestamps].sort();
   const pendingWithdrawals: PendingWithdrawal[] = [];
 
-  let referenceTimestamp = 0;
+  let referenceTimestamp = 0n;
   for (const index in sortedTimestamps) {
     const timestamp = sortedTimestamps[index];
     const amount = values[index];
 
     /* If next timestamp is not within allowed time span create new pending withdrawal */
-    if (parseInt(timestamp) > referenceTimestamp + groupingTimeSpan) {
-      referenceTimestamp = parseInt(timestamp);
+    if (timestamp > referenceTimestamp + groupingTimeSpan) {
+      referenceTimestamp = timestamp;
       pendingWithdrawals.push({
         amount: new Celo(amount),
-        timestamp,
+        timestamp: timestamp.toString(),
       });
       continue;
     }
 
     /* If next timestamp is within allowed time span merge it with the last pending withdrawal */
     const lastPendingWithdrawal = pendingWithdrawals[pendingWithdrawals.length - 1];
-    lastPendingWithdrawal.timestamp = timestamp;
-    lastPendingWithdrawal.amount = new Celo(lastPendingWithdrawal.amount.plus(amount));
+    lastPendingWithdrawal.timestamp = timestamp.toString();
+    lastPendingWithdrawal.amount = new Celo(lastPendingWithdrawal.amount.plus(amount as any));
   }
 
   return pendingWithdrawals.reverse();
 };
 
 const pendingWithdrawalsLoadInterval = 60 * 1000;
 
-export const useWithdrawals = (address: string | null) => {
+export const useWithdrawals = (address: `0x${string}` | undefined) => {
```

Comment:
"`0x${string}` | undefined" i see this used a few places it be beneficial to create a `type OptionalAddress`